Lib/gzip.py

```diff
@@ -583,8 +583,7 @@ def main():
                 g = sys.stdout.buffer
             else:
                 if arg[-3:] != ".gz":
-                    print("filename doesn't end in .gz:", repr(arg))
-                    continue
+                    sys.exit("filename doesn't end in .gz:", repr(arg))
```

Comment:
Shouldn’t this use string concatenation or f-string to pass one argument to sys.exit?

Signature is `sys.exit(status=None)`

Reply:
You are right! We can not pass two strings to sys.exit().
I don't know why this passed the test.

Reply:
Now I got it. The stderr is:

```
Traceback (most recent call last):
(snip)
  File "/Users/inada-n/work/python/cpython/Lib/gzip.py", line 586, in main
    sys.exit("filename doesn't end in .gz:", repr(arg))
TypeError: exit expected at most 1 argument, got 2
```

And it passes the test `self.assertIn(b"filename doesn't end in .gz:", err)`.

Reply:
Fixed in #24652